### PR TITLE
Attempt to fix subrepo push

### PR DESCRIPTION
**Pull request by pokey**
Looks like we need to set a username and email for git before we can push

### CODE REVIEW

**Review by AndreasArvidsson — APPROVED** [review]

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -52,5 +52,7 @@ jobs:
           path: git-subrepo
       - run: |
           source ../git-subrepo/.rc
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```
Comment (auscompgeek) [inline .github/workflows/deploy.yaml:56]:
```suggestion
          git config user.email action@github.com
```
I believe is the email for @actions-user

Comment (pokey) [reply .github/workflows/deploy.yaml:56]: I just was following https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

Comment (pokey) [reply .github/workflows/deploy.yaml:56]: maybe that's out of date?  I wouldn't be shocked 😅

Comment (auscompgeek) [reply .github/workflows/deploy.yaml:56]: :shrug:

**Pull request merged**